google/cloud/forseti/common/gcp_api/cloudasset.py

```diff
@@ -0,0 +1,303 @@
+            repository = self.repository.organizations
+        else:
+            raise ValueError('parent must start with either projects/ or '
+                             'organizations/')
+
+        try:
+            results = repository.export_assets(
+                parent, destination_object, content_type=content_type,
+                asset_types=asset_types)
+            if blocking:
+                results = self.wait_for_completion(parent, results,
+                                                   timeout=timeout)
+        except (errors.HttpError, HttpLib2Error) as e:
+            LOGGER.error('Error exporting assets for parent %s: %s', parent, e)
+            raise api_errors.ApiExecutionError(parent, e)
+        except api_errors.OperationTimeoutError as e:
+            LOGGER.warn('Timeout exporting assets for parent %s: %s', parent, e)
+            raise
+
+        LOGGER.info('Exporting assets for parent %s. Result: %s',
+                    parent, results)
+        return results
+
+    def get_operation(self, operation_name):
+        """Get the Operations Status.
+
+        Args:
+            operation_name (str): The name of the operation to get.
+
+        Returns:
+            dict: Operation status and info.
+
+        Raises:
+            ApiExecutionError: Returns if there is an error in the API response.
+            ValueError: Raised on invalid parent resource name.
+        """
+        if operation_name.startswith('projects/'):
+            repository = self.repository.projects_operations
+        elif operation_name.startswith('organizations/'):
+            repository = self.repository.organizations_operations
+        else:
+            raise ValueError('operation_name must start with either projects/ '
+                             'or organizations/')
+        try:
+            results = repository.get(operation_name)
+            LOGGER.debug('Getting the operation status, operation_name = %s, '
+                         'results = %s', operation_name, results)
+        except (errors.HttpError, HttpLib2Error) as e:
+            raise api_errors.ApiExecutionError(operation_name, e)
+
+        return results
+
+    def wait_for_completion(self, parent, operation, timeout=0,
+                            initial_delay=None, retry_delay=None):
+        """Wait for the operation to complete.
+
+        Args:
+            parent (str): The name of the parent resource to export assests
+                under.
+            operation (dict): The operation response from an API call.
+            timeout (float): The maximum time to wait for the operation to
+                complete.
+            initial_delay (float): The time to wait before first checking if the
+                API has completed. If None then the default value is used.
+            retry_delay (float): The time to wait between checking if the
+                API has completed. If None then the default value is used.
```

Comment:
It is not clear here, what is the default value that is being referred to.  Can we please clarify this?